### PR TITLE
fix table colour in project resource page

### PR DESCRIPTION
**Pull request by poornimashah**
Targets #3019 

### Checklist:

- [x] I have performed a self-review of my own code.
feature/3019/fix-table-colour

### CODE REVIEW

**Comment by rahulpanwar16** [comment]
@poornimashah There are a few suggestions when you are making the PR.

```In Assignees``` - Assign yourself before assigning to the reviewers.
```In Labels``` - Put the label ```ready for review``` before assigning it to the reviewers.
```In Reviewers``` - Assign ```Gaurav Gusain``` in the PR Reviewers and please remove other all names from reviewers. 
```In Targets``` - Write issue-id  i.e #3019 in your case.
```In PR Name``` - Write down the meaningful PR name i.e fix table colour on project resource page.

**Comment by Ayush8923** [comment]
@poornimashah Please fix the CI checks. Then assign reviewers to review the PR.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please revert the `package-lock.json` file from this PR.

File: Modules/Project/Resources/views/resource-requirement.blade.php [hunk 1]
```diff
@@ -23,13 +23,13 @@
         <br>
 
         <div>
-            <table class="table table-bordered table-striped">
+            <table class="table table-bordered table-striped" >
```
Comment (Ayush8923) [inline Modules/Project/Resources/views/resource-requirement.blade.php:26]:
```suggestion
            <table class="table table-bordered table-striped">
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
One suggested change. Please remove the extra spaces.

File: Modules/Project/Resources/views/resource-requirement.blade.php (hunk 1 above)
Comment (Ayush8923) [inline Modules/Project/Resources/views/resource-requirement.blade.php:26]:
```suggestion
            <table class="table table-bordered table-striped">
```

**Review by Ayush8923 — COMMENTED** [review]
Approved with one comment. We don't need to this extra small space.

**Review by Ayush8923 — APPROVED** [review]

**Review by Ayush8923 — APPROVED** [review]

File: Modules/Project/Resources/views/resource-requirement.blade.php (hunk 1 above)
Comment (poornimashah) [reply Modules/Project/Resources/views/resource-requirement.blade.php]: done sir

**Pull request merged**